models/events.js

```diff
@@ -237,11 +237,46 @@ const createEventCode = async (eventCodeData) => {
   }
 };
 
+const getEventById = async ({ id }) => {
```

Comment:
eventSnapshotData variable is not being checked for null before it is returned. This means that if the event with the given ID does not exist, the eventSnapshotData variable will be null and the function will return null.

Reply:
Yes it will return null but if you will see the ln 255 I have added a check for if eventData is null then it will throw an error.